src/PDPVerifier.sol

```diff
@@ -599,23 +640,18 @@ contract PDPVerifier is Initializable, UUPSUpgradeable, OwnableUpgradeable {
 
     function calculateProofFee(uint256 setId, uint256 estimatedGasFee) public view returns (uint256) {
         uint256 rawSize = 32 * challengeRange[setId];
-        (uint64 filUsdPrice, int32 filUsdPriceExpo) = getFILUSDPrice();
```

Comment:
can we now remove the other usd oracling code or is it used elsewhere?

Reply:
afaik, it is not used for anything else